### PR TITLE
[H5R File article, Native File System article] Add pointers to new features

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add pointer to Native File System API article to H5R File article.
- Add pointer to browser-nativefs to Native File System article.

### CODE REVIEW

File: src/site/content/en/blog/native-file-system/index.md [hunk 1]
```diff
@@ -262,6 +262,13 @@ doesn't match the spec:
 * Writing to a file using a [`WritableStream`][writablestream].
 * The [`FileSystemDirectoryHandle.resolve()`][fs-dir-handle] method.
 
+{% Aside 'note' %}
+  Since the API is not compatible with all browsers yet,
+  we provide a library called
+  [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs)
+  that uses the new API wherever possible, and else falls back to the legacy approaches.
```
Comment (jpmedley) [inline src/site/content/en/blog/native-file-system/index.md]:
```suggestion
  that uses the new API wherever it is available, but falls back to legacy approaches when it is not.
```

File: src/site/content/en/blog/read-files/index.md [hunk 2]
```diff
@@ -20,6 +20,15 @@ This guide shows you how to:
 * [Slice a file's content](#slice)
 * [Monitor the progress of a file read](#monitor)
 
+{% Aside 'note' %}
+  The [Native File System API](/native-file-system/) is the modern alternative
+  to many of the operations listed in the present article.
```
Comment (jpmedley) [inline src/site/content/en/blog/read-files/index.md]:
```suggestion
  to many of the operations listed in this article.
```

File: src/site/content/en/blog/read-files/index.md [hunk 3]
```diff
@@ -20,6 +20,15 @@ This guide shows you how to:
...
+  Since the API is not compatible with all browsers yet,
+  we provide a library called
+  [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs)
+  that uses the new API wherever possible, and else falls back to the legacy approaches.
```
Comment (jpmedley) [inline src/site/content/en/blog/read-files/index.md]:
```suggestion
  that uses the new API wherever it is available, but falls back to legacy approaches when it is not.
```

File: src/site/content/en/blog/read-files/index.md [hunk 4]
```diff
@@ -261,4 +270,4 @@ Browser compatibility data for the APIs used in this section:
 [text]: https://developer.mozilla.org/docs/Web/API/FileReader/readAsText
 [dropeffect]: https://developer.mozilla.org/docs/Web/API/DataTransfer/dropEffect
 [propagation]: https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Examples#Example_5:_Event_Propagation
-[preventdefault]: https://developer.mozilla.org/docs/Web/API/Event/preventDefault
\ No newline at end of file
+[preventdefault]: https://developer.mozilla.org/docs/Web/API/Event/preventDefault
```
Comment (jpmedley) [inline src/site/content/en/blog/read-files/index.md]: I don't see what's different here. Was there supposed to be a change?

Comment (tomayac) [reply src/site/content/en/blog/read-files/index.md]: My editor removed a trailing space.

**Pull request merged**